### PR TITLE
fixes #7567 - update syntax of deprecated find_by_[attribute] dynamic methods for rails 4

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

**Comment by isratrade** [comment]
@unorthodoxgeek, rebase and dom's commit 'fixes #8735 - enable config drive on Rackspace servers' should go away

File: test/unit/user_test.rb [hunk 1]
```diff
@@ -656,6 +656,117 @@ def editing_self_helper
       @options.merge!({ :controller => "hosts", :id => User.current.id })
       refute User.current.editing_self?(@options)
     end
+=======
```
Comment (unorthodoxgeek) [inline test/unit/user_test.rb]: woah! you forgot to solve this conflict...

Comment (unorthodoxgeek) [inline test/unit/user_test.rb]: @isratrade

Comment (isratrade) [inline test/unit/user_test.rb]: @unorthodoxgeek, fixed

**Comment by unorthodoxgeek** [comment]
thanks @isratrade :+1:

**Pull request merged**